nrfs/src/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+
+if((NOT CONFIG_SOC_NRF54H20_CPUSYS) AND (NOT CONFIG_SOC_NRF9230_CPUSYS))
+    zephyr_sources_ifdef(CONFIG_TEMP_SERVICE_ENABLED services/nrfs_temp.c)
+    zephyr_sources_ifdef(CONFIG_MRAM_SERVICE_ENABLED services/nrfs_mram.c)
+    zephyr_sources_ifdef(CONFIG_RESET_SERVICE_ENABLED services/nrfs_reset.c)
+    zephyr_sources_ifdef(CONFIG_USB_SERVICE_ENABLED services/nrfs_usb.c)
+    zephyr_sources_ifdef(CONFIG_PMIC_SERVICE_ENABLED services/nrfs_pmic.c)
+    zephyr_sources_ifdef(CONFIG_DVFS_SERVICE_ENABLED services/nrfs_dvfs.c)
+    zephyr_sources_ifdef(CONFIG_DIAG_SERVICE_ENABLED services/nrfs_diag.c)
+    zephyr_sources_ifdef(CONFIG_CLOCK_SERVICE_ENABLED services/nrfs_clock.c)
```

Comment:
these Kconfigs need proper namespacing

Reply:
What do you mean by "proper namespacing"?

Reply:
`CONFIG_NRFS`